peak_finder/driver.py

```diff
@@ -277,18 +287,16 @@ def run(self):  # noqa  # pylint: disable=R0912, R0914, too-many-statements
             print("Exporting . . .")
             group_points = None
             if group_center:
-                channel_group = np.hstack(channel_group)  # Start count at 1
-
                 # Create reference values and color_map
                 group_map, color_map = {0: "Unknown"}, [[0, 0, 0, 0, 0]]
                 for ind, (name, group) in enumerate(channel_groups.items()):
                     group_map[ind + 1] = name
                     color_map += [[ind + 1] + hex_to_rgb(group["color"]) + [0]]
 
-                color_map = np.core.records.fromarrays(
-                    np.vstack(color_map).T,
-                    names=["Value", "Red", "Green", "Blue", "Alpha"],
-                )
+                # color_map = np.rec.fromarrays(
+                #     np.vstack(color_map).T,
+                #     names=["Value", "Red", "Green", "Blue", "Alpha"],
+                # )
```

Comment:
**question:** something broken?

to be fixed in another issue?

please, either remove the code commented out, or leave a comment in code as to why it was commented out